test_scripts/API/RAI/ideas_RegisterAppInterface_parameter_steeringWheelLocation.txt

```diff
@@ -0,0 +1,58 @@
+./test_scripts/API/RAI/001_ATF_N_HMI_Does_Not_Respond_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_HMI_capabilities_HMI_doesn't_respond_UI.GetCapabilities

```

Comment:
@OVikhrov, I agree with you, but we still don't have final organisation of scripts and test sets. I will do this when we have correct structure. For now will leave them in this Test Set, not to be lost.

According to "Also, I am do not sure that "new line" separation in test set list will be correctly interpreted by CI." - I don't have enough knowledge for this. 
This kind of check is mandatory for invalid parameter of SDL and current scripts.